src/assets/style/_layout.scss

```diff
@@ -48,53 +49,109 @@
 // --------------------------------------
 
 //
-// sidenav (not used anymore...? 2019-03-18)
-.sidenav-container {
-  .sidenav {
-    background-color: lighten($primary, 50%);
-    letter-spacing: 0.3px;
-    width: 36px;
-
-    &.wide {
-      width: 240px;
-    }
+// sidenav in project view
+.project-view {
+  height: calc(100vh - #{$header-height});
 
-    &.narrow {
-      width: 36px;
+  .project-nav {
+    width: 220px;
+
+    .mat-title,
+    .mat-subheading-2,
+    .mat-subheading-1 {
+      margin: 0;
     }
 
-    .mat-drawer-inner-container {
-      overflow-x: hidden;
+    .mat-list-item {
+      height: 64px;
 
-      .mat-list {
-        padding-top: 0;
+      &:hover {
+        background: rgba(0, 0, 0, .04);
       }
     }
 
-    .side-nav-footer {
-      border-top: 1px solid rgba(0, 0, 0, 0.12);
-      bottom: 0;
-      height: 36px;
-      position: absolute;
-      width: 100%;
+    .mat-expansion-panel {
+      box-shadow: none !important;
 
-      .mat-button {
-        min-width: 30px;
-        padding: 0;
-        width: 100%;
+      .mat-expansion-panel-header {
+        padding: 0 16px !important;
+        // display: block !important;
+      }
 
-        .mat-icon {
-          margin: 0 auto;
+      .sidenav-panel-header{
+        width: 100%;
+        mat-panel-title {
+          overflow: hidden;
+          white-space: nowrap;
+          text-overflow: ellipsis;
+          width: 100%;
+          display: block;
         }
       }
+
+      // reset default value, if the panel is expanded
+      &.mat-expanded mat-panel-title {
+        overflow: unset;
+        white-space: inherit;
+        text-overflow: unset;
+        display: flex;
+      }
+
+      .mat-expansion-panel-body {
+        padding: 0 !important
+      }
     }
   }
 
-  .mat-sidenav-content {
-    background: #fff;
-  }
 }
 
+
+// .sidenav-container {
```

Comment:
It was still there from a previous sidenav concept. It has some good configurations but I had to comment it out to avoid conflicts with the new sidenav concept. I'll move it to the component styling sheet to re-activate the good stuff in the next iteration.

Reply:
resolved in 190f1f28